java/KafkaConfigProviders/Kafka-mTLS-Keystore-Sql-ConfigProviders/README.md

```diff
@@ -0,0 +1,105 @@
+## Sample illustrating how to use MSK config providers in Flink Kafka connectors, for mTLS authentication
+
+* Flink version: 1.20
+* Flink API: Table API & SQL
+* Language: Java (11)
+* Flink connectors: Kafka (mTLS authentication)
+
+This sample illustrates how to configure the Flink Kafka connectors (KafkaSource and KafkaSink) 
+retrieving custom KeyStore at runtime, using Config Providers.
+More details on the MSK Config Providers in [this repo](https://github.com/aws-samples/msk-config-providers).
+
+* KeyStore are fetched from S3, when the job starts.
+* The password to open KeyStore is also fetched when the job starts, from AWS Secret Manager.
+* No secret is packaged with the application.
+
+### High level approach
+
+NOTE: This repo already includes a previously built version of the msk-config-providers JAR from [this repo](https://github.com/aws-samples/msk-config-providers).
+
+1. Build this repo using `mvn clean package`.
+2. Setup Flink app using the jar from the build above. Please follow the instructions [here](https://docs.aws.amazon.com/managed-flink/latest/java/getting-started.html).
+3. Please ensure that you specify appropriate values for the application properties (S3 location, secrets manager key, etc.).
+
+Also look at the [troubleshooting guide](docs/troubleshoot-guide.md) for solving common issues during the setup. 
+
+
+### Configuring the Kafka connector
+
+The following snippet shows how to configure the service providers and other relevant properties.
+
+See [StreamingJob.java](src/main/java/com/amazonaws/services/msf/StreamingJob.java):
+
+```java
```

Comment:
This is the snipped from DataStream. It should be the snipped from the Table/SQL example instead

Reply:
Updated snippet for Table Sql